projects/ng-kaart/src/lib/kaart/feature-tabel/feature-tabel-data.component.html

```diff
@@ -57,9 +57,14 @@
                 matTooltip="Zoom naar locatie"
                 >{{ row.velden[header.name].maybeValue.getOrElse("") }}</span
               >
-              <span *ngIf="!first">{{
-                row.velden[header.name].maybeValue.getOrElse("")
-              }}</span>
+              <span *ngIf="!first && row.velden[header.name].maybeLink.isNone()">
+                  {{ row.velden[header.name].maybeValue.getOrElse("") }}
+              </span>
+              <span *ngIf="!first && row.velden[header.name].maybeLink.isSome()">
+                  <a href="{{ row.velden[header.name].maybeLink.getOrElse() }}" target="_blank">
```

Comment:
"Argument types do not match parameters"
toNullable lijkt me hier beter passen
```suggestion
                  <a href="{{ row.velden[header.name].maybeLink.toNullable() }}" target="_blank">
```